### DISCUSSION TITLE
Can I download playlists?

### DISCUSSION THREAD

**Discussion by Miqueas** [NickvisionApps/Parabolic#737] [Q&A] [answered]
I tried to download playlists a while ago and didn't worked.

**Comment by nlogozzo** [comment]
What was the link of the playlist you tried?
YouTube playlists should have no issues...

**Reply by Miqueas** [reply]
tbh I don't remember, but I tried a while ago and didn't worked, never tried again

**Reply by nlogozzo** [reply, marked as answer]
Well that doesn't really help advance the issue 😅

If you try it again and run into the same issue feel free to come back and comment with the URL you tried.

**Reply by Miqueas** [reply]
You're right. I will try again later and comment here if something went wrong.
